configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_hrnet-w32_fp16-8xb64-210e_coco-256x192.py

```diff
@@ -1,4 +1,4 @@
-_base_ = ['./hrnet_w32_coco_256x192.py']
+_base_ = ['./td-hm_hrnet-w32_8xb64-210e_coco-256x192.py']
```

Comment:
`fp16` is deprecated in configs. We should use `AmpOptimWrapper` for automatic mixed precision training. Please refer to the `amp` argument in `tools/train.py`.